translate/cloud-client/translate_v3_get_supported_languages_with_target.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud import translate
+
+
+def get_supported_languages_with_target(
+    language_code="YOUR_TARGET_LANG_CODE", project_id="YOUR_PROJECT_ID"
```

Comment:
Should `language_code` just be inlined and provide a link to supported languages?

Reply:
Yea, that makes sense, and I will add the link too